frontend/layouts/UnsupportedScreenSize/_styles.scss

```diff
@@ -9,32 +9,31 @@
   align-items: center;
   justify-content: center;
   gap: $pad-medium;
-  
```

Comment:
To test host count cards on low widths, add `display: none;` on `line 12` in this file and comment out`/CoreLayout/_styles.scss` `line 18`